assets/js/modules/analytics/components/common/AccountCreate/index.js

```diff
@@ -171,6 +174,23 @@ export default function AccountCreate() {
 				{ ! hasProvisioningScope && <span>{ __( 'You will need to give Site Kit permission to create an Analytics account on your behalf and also accept the Google Analytics terms of service.', 'google-site-kit' ) }</span> }
 			</p>
 
+			{ isGA4enabled && (
+				<SettingsNotice
```

Comment:
@danielgent, please, use the `GA4PropertyNotice` component instead.

Reply:
@eugene-manuilov the ticket does explicitly say `use the SettingsNotice component` though!